docs/source/index.rst

```diff
@@ -0,0 +1,291 @@
+cluster_generator
+=================
+
+|yt-project| |docs| |testing| |Github Page| |Pylint| |coverage| |ncodes|
+
+.. raw:: html
+
+   <hr style="height:10px;background-color:black">
+
+The `Cluster Generator Project <https://eliza-diggins.github.io/cluster_generator>`_ (CGP) is a cross-platform Python library for generating initial conditions of galaxy clusters for N-body / hydrodynamics codes.
+CGP provides a variety of construction approaches, different physical assumption, profiles, and gravitational theories. Furthermore, the CGP is intented to interface with
+the vast majority of N-body / hydrodynamics codes, reducing the headache of converting initial conditions between formats for different simulation softwares. GCP's goal is to provide
```

Comment:
This description is biting off more than we can chew, I think. We should just say "a number of N-body / hydrodynamics codes used in studies of galaxy clusters."

Reply:
I agree! Definitely a bit over confident on my part. I've revised this.